autogpt_platform/backend/backend/data/execution.py

```diff
@@ -115,6 +116,10 @@ def error_rate(self) -> float:
         ExecutionStatus.QUEUED,
         ExecutionStatus.RUNNING,
     ],
+    ExecutionStatus.WAITING_FOR_REVIEW: [
+        ExecutionStatus.RUNNING,
+        ExecutionStatus.TERMINATED,  # Allow terminating execution during review
```

Comment:
which reminds me that means we need to set the status of waiting_for_review (not only running status) into terminated when we are stopping the graph execution